worker/orca_grader/docker_utils/images/image_loading.py

```diff
@@ -23,8 +25,10 @@ def load_image_from_tgz(tgz_file_path: str) -> None:
     ]
     _LOGGER.debug(f"Attempting to load image from file {tgz_file_path}")
     result = subprocess.run(program_args, check=True, capture_output=True)
-    _LOGGER.debug(f"Docker save stdout: {result.stdout.decode() if result.stdout is not None else '<None>' }")
-    _LOGGER.debug(f"Docker save stderr: {result.stderr.decode() if result.stderr is not None else '<None>' }")
+    _LOGGER.debug(
```

Comment:
I'm still concerned that these logging statements won't run if the subprocess throws an error.  So, stick them in a try/catch block: as in
```
try
  result = subprocess.run(...)
  ...log stdout and stderr
  ...log("Image loaded")
catch e
  log("Attempt to load image failed: " + e.exit code, e.stdout, e.stderr)
  rethrow e
```